pkg/processresetqueue/reset.go

```diff
@@ -169,7 +169,7 @@ func Reset(input *ResetInput) (*ResetOutput, error) {
 
 // extractRoleNameFromARN returns the name of the role from its arn
 func extractRoleNameFromARN(arn string) (string, error) {
-	reg := regexp.MustCompile("arn:aws:iam::\\d{12}:role/(.+)")
+	reg := regexp.MustCompile(`arn:aws:iam::\d{12}:role/(.+)`)
```

Comment:
linter says "use backticks so you don't have to double escape regexes"

I added a basic unit test around this method, to make sure I didn't break anything